src/structures/GuildEmoji.js

```diff
@@ -67,7 +68,7 @@ class GuildEmoji extends BaseGuildEmoji {
       .guilds(this.guild.id)
       .emojis(this.id)
       .get()
-      .then(emoji => this.client.users.add(emoji.user));
+      .then(emoji => this._patch(emoji) || this.author);
```

Comment:
The usage of `||` here is not optimal.
If we decided to have `_patch` return the value as it probably is intended to do (just ignored by us everywhere), it would return the emoji and not the user.

If anything, this would be a place for the `,` operator, but please don't use it.

This was already suggested in #library-discussion by Papaia#4254, but it did not made it into a review here yet, so here I go:
I think it would be better to transform this method to an async method similar to
https://github.com/discordjs/discord.js/blob/b0b62d63cf5545034a65bcf7ce0d80d2bf716354/src/structures/User.js#L270-L275

You could also actually `throw` the errors above then, instead of wrapping them in a rejected promise.

Reply:
Alright, I will commit my changes.